pages/agent/v3.md.erb

```diff
@@ -124,6 +124,26 @@ After repository checkout, resolve `BUILDKITE_COMMIT` to a commit hash. This mak
   <p>To use it, set <code>experiment="resolve-commit-after-checkout"</code> in your <a href="/docs/agent/v3/configuration#experiment"> agent configuration</a>.</p>
 </div>
 
+### Flock File Locks
+
+Changes the file lock implementation from github.com/nightlyone/lockfile to github.com/gofrs/flock to address an issue where file locks are never released by agents that don't shut down cleanly. The new file locks are implemented at the kernel level, and are aware of when their parent process dies.
```

Comment:
Not suuuuuper easily... in this case it means that the OS killed it, rather than politely asking it to stop please. I think in context, our readers will understand what we mean.

Reply:
Okay then 👍